packages/backend/src/config/app.ts

```diff
@@ -98,6 +98,11 @@ export const Config = {
   signatureSecret: process.env.SIGNATURE_SECRET, // optional
   signatureVersion: envInt('SIGNATURE_VERSION', 1),
 
+  openPaymentsSpec: envString(
+    'OPEN_PAYMENTS_SPEC',
+    'https://github.com/interledger/open-payments/raw/3930448672cfc678ec2bc02938566a316d83871c/open-api-spec.yaml'
```

Comment:
Does that need to be updated when the spec changes?

Reply:
Yes / as Rafiki's functionality to match it.

Reply:
Is there a way that this could be automated? On the other hand, the spec should be fairly stable pretty soon :thinking:

Reply:
It could use https://raw.githubusercontent.com/interledger/open-payments/master/open-api-spec.yaml
but I'd prefer pinning to a known compatible commit.